### ISSUE TITLE
Sending RC over MAVLink

### ISSUE THREAD

**Issue by superjax** [BYU-MAGICC/fcu_io2#12]
@dpkoch, I copied your code you use to send servo raw and modified to it send the rc signal.  It's working, except that for some reason the ack callback is getting run in fcu_io2.  I don't really know how it works (the boost pointer structure is pretty confusing.  It's hard to know where stuff is happening).  Anyway, I was wondering if you could look at it and help me figure out why that ack function is getting called over and over.

**Comment by dpkoch** [comment]
I commented on your commit, but not sure if you got it. You're missing a `break` statement at the end of the case you added.

Also, yeah I don't love how things are structured now; I've been wanting to redo it for awhile. As you now know, it's kind of a pain to add support for a new message.

The reason I went with the boost function stuff was so I could have a core library that was independent of the ROS wrapper I chose to put around it, so that someone else could easily make their own wrapper. I think I still stand by that decision, but it definitely needs to be streamlined a bit. I've been thinking it over in the back of my mind; someday I'll actually get around to it.
